install.sh

```diff
@@ -3,10 +3,11 @@
 set -o errexit
 
 USAGE_TEXT="\
-Usage: $0 [-adh]
+Usage: $0 [-adsh]
 
   -a                          automated install, no questions
   -d                          install development dependencies
+  -s                          skip python package installation and just do pip install
```

Comment:
I'm wondering if it would be good to add this to `Install.ps1` as well.  I'm not entirely sure.  On the one hand, it is pointless right now afaik, on the other, maintaining parity in the accepted options is useful.

Reply:
As to `Install.ps1`, it does not try to install `python.exe`.
So by default `-s` is `true`.
Windows user should install Python by themselves, so it doesn't require Administrative previledge.